.github/workflows/vib-redis.yaml

```diff
@@ -0,0 +1,71 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+    types: 
+      - opened 
+      - reopened
```

Comment:
We should include "synchronize" that means "run the action if a new commit arrives":
- ref: https://frontside.com/blog/2020-05-26-github-actions-pull_request/